jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -120,8 +120,12 @@ def on_mouse_or_key_event(self, data):
             self.label_mouseover.pixel = (fmt.format(x, y))
 
             if coords_status:
-                coo = image.coords.pixel_to_world(x, y).icrs
-                self.label_mouseover.set_coords(coo)
+                try:
+                    coo = image.coords.pixel_to_world(x, y).icrs
+                except Exception:  # WCS might not be celestial
+                    self.label_mouseover.reset_coords_display()
```

Comment:
is catching all exceptions intentional here?  Will this ever make it difficult to discover/debug an underlying bug in `pixel_to_world`?

Reply:
I'm on the fence.  On one hand, users reporting bugs is better than them going unnoticed, but they might report anyways since the display will be blank?  Writing to the debug snackbar logger (now that we have easier control over logger levels) might also be an option - but I'm ok with waiting until we run into a case where it fails to implement that.